source/common/runtime/runtime_features.cc

```diff
@@ -82,6 +82,12 @@ ABSL_FLAG(uint64_t, re2_max_program_size_warn_level,            // NOLINT
 namespace Envoy {
 namespace Runtime {
 
+bool hasRuntimePrefix(absl::string_view feature) {
+  return (absl::StartsWith(feature, "envoy.reloadable_features.") &&
+          !absl::StartsWith(feature, "envoy.reloadable_features.FLAGS_quic")) ||
```

Comment:
Why this carve out?

Reply:
QUICHE flags aren't a part of the runtime flag framework below so need to be exempted.  commented in code.